examples/imx500/imx500_object_detection_demo.py

```diff
@@ -121,24 +120,47 @@ def get_args():
                         default=None, help="Run post process of type")
     parser.add_argument("-r", "--preserve-aspect-ratio", action="store_true",
                         help="preprocess the image with  preserve aspect ratio")
-    parser.add_argument("--labels", type=str, default="assets/coco_labels.txt",
+    parser.add_argument("--labels", type=str,
                         help="Path to the labels file")
+    parser.add_argument("--print-intrinsics", action="store_true",
+                        help="Print JSON network_intrinsics then exit")
     return parser.parse_args()
 
-
 if __name__ == "__main__":
     args = get_args()
 
     # This must be called before instantiation of Picamera2
     imx500 = IMX500(args.model)
+    intrinsics = imx500.intrinsics
```

Comment:
I think I would prefer ``imx500.network_intrinsics``, but not a big deal.

Reply:
I'll make the change.